.github/workflows/save_atomic_files.yml

```diff
@@ -0,0 +1,38 @@
+name: Save atomic files every week
+
+on: 
+  schedule:
+    - cron: '0 0 * * 7' # run at midnight every Sunday
+
+
+jobs:
+  create_artifacts:
+    uses: ./.github/workflows/bridge.yml
+  
+  zip_artifacts:
+    needs: create_artifacts
+    if: always()
```

Comment:
What does this do? Does it mean that this runs even if the previous step fails? If this is the case, I don't think we want that?

Reply:
Currently, the previous step would fail because the reference data doesn't match. If I don't add this step, we won't be able to zip files because this step won't run.

Reply:
GitHub secrets cannot be accessed in pull requests, which is why there is no way to verify this pull request. But I tried [running](https://github.com/atharva-2001/carsus_atomic_file_search/actions/runs/4015876053/jobs/6898155789) a similar [workflow](https://github.com/atharva-2001/carsus_atomic_file_search/blob/main/.github/workflows/search.yml) on my own repo and that worked.

Reply:
Concerning the `if: always()`: I guess it is fine as is, but isn't it possible to do the if statement based on whether, for example, the "Compare Reference Data" step in the bridge workflow has run?